src/ResourceManager/Sql/Commands.Sql/Auditing/Cmdlet/GetAzureSqlDatabaseAuditingPolicy.cs

```diff
@@ -21,6 +22,7 @@ namespace Microsoft.Azure.Commands.Sql.Auditing.Cmdlet
     /// Returns the auditing policy of a specific database.
     /// </summary>
     [Cmdlet(VerbsCommon.Get, "AzureRmSqlDatabaseAuditingPolicy", SupportsShouldProcess = true), OutputType(typeof (AuditingPolicyModel))]
+    [Obsolete]
```

Comment:
something like `[Obsolete("blah blah blah")]` describing why the cmdlet is deprecated and what to use instead